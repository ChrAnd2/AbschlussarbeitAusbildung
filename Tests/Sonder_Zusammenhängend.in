//***********************************************
//+ Sonder_Zusammenhängend	     Viele Start- und Endvorgänge, zwei Netzpläne die nur über einen Vorgang verbunden sind.
//***********************************************
//Vorgangsnummer; Vorgangsbezeichnung; Dauer; Vorgänger; Nachfolger
1; A1; 1; -; 7
2; A2; 1; -; 8
3; A3; 1; -; 8
4; A4; 1; -; 11
5; A5; 1; -; 9
6; A6; 1; -; 16
7; A; 1; 1; 10
8; B; 1; 2,3; 10,11
9; C; 1; 5; 11,15,16
10; D; 1; 7,8; 12
11; E; 1; 8,4,9; 13,14
12; E1; 1; 10; -
13; E2; 1; 11; -
14; E3; 1; 11; -
15; E4; 1; 9; -
16; E5; 1; 9,6; -