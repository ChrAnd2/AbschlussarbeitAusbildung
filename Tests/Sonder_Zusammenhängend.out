Sonder_Zusammenhängend	     Viele Start- und Endvorgänge, zwei Netzpläne die nur über einen Vorgang verbunden sind.

Vorgangsnummer; Vorgangsbezeichnung; D; FAZ; FEZ; SAZ; SEZ; GP; FP
1; A1; 1.0; 0.0; 1.0; 0.0; 1.0; 0.0; 0.0
2; A2; 1.0; 0.0; 1.0; 0.0; 1.0; 0.0; 0.0
3; A3; 1.0; 0.0; 1.0; 0.0; 1.0; 0.0; 0.0
4; A4; 1.0; 0.0; 1.0; 1.0; 2.0; 1.0; 1.0
5; A5; 1.0; 0.0; 1.0; 0.0; 1.0; 0.0; 0.0
6; A6; 1.0; 0.0; 1.0; 1.0; 2.0; 1.0; 1.0
7; A; 1.0; 1.0; 2.0; 1.0; 2.0; 0.0; 0.0
8; B; 1.0; 1.0; 2.0; 1.0; 2.0; 0.0; 0.0
9; C; 1.0; 1.0; 2.0; 1.0; 2.0; 0.0; 0.0
10; D; 1.0; 2.0; 3.0; 2.0; 3.0; 0.0; 0.0
11; E; 1.0; 2.0; 3.0; 2.0; 3.0; 0.0; 0.0
12; E1; 1.0; 3.0; 4.0; 3.0; 4.0; 0.0; 0.0
13; E2; 1.0; 3.0; 4.0; 3.0; 4.0; 0.0; 0.0
14; E3; 1.0; 3.0; 4.0; 3.0; 4.0; 0.0; 0.0
15; E4; 1.0; 2.0; 3.0; 2.0; 3.0; 0.0; 0.0
16; E5; 1.0; 2.0; 3.0; 2.0; 3.0; 0.0; 0.0

Anfangsvorgang: 1,2,3,4,5,6
Endvorgang: 12,13,14,15,16
Maximale Gesamtdauer (da mehrere Endzustaende): 4.0

Kritische Pfade:
1. 1->7->10->12
2. 2->8->10->12
3. 2->8->11->13
4. 2->8->11->14
5. 3->8->10->12
6. 3->8->11->13
7. 3->8->11->14
8. 5->9->11->13
9. 5->9->11->14
10. 5->9->15
11. 5->9->16
